features/Reisdocumenten/raadpleeg-met-reisdocumentnummer/opschorting-bijhouding-gba.feature

```diff
@@ -0,0 +1,114 @@
+#language: nl
+
+@gba
+Functionaliteit: RaadpleegMetReisdocumentnummer van persoonslijst met opschorting bijhouding
+
+  Rule: Een reisdocument op een persoonslijst met reden opschorting bijhouding "W" (wissen) wordt niet geleverd
+
+    Abstract Scenario: persoonslijst heeft opschorting bijhouding reden "W" en <fields> van het reisdocument wordt gevraagd
+      Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende 'inschrijving' gegevens
+      | datum opschorting bijhouding (67.10) | reden opschorting bijhouding (67.20) |
+      | 20220829                             | W                                    |
+      En de persoon heeft een 'reisdocument' met de volgende gegevens
+      | naam                        | waarde    |
+      | soort reisdocument (35.10)  | PN        |
+      | nummer reisdocument (35.20) | NE3663258 |
+      Als gba reisdocumenten wordt gezocht met de volgende parameters
+      | naam               | waarde                         |
+      | type               | RaadpleegMetReisdocumentnummer |
+      | reisdocumentnummer | NE3663258                      |
+      | fields             | <fields>                       |
+      Dan heeft de response 0 reisdocumenten
+
+      Voorbeelden:
+      | fields                     |
+      | reisdocumentnummer         |
+      | soort                      |
+      | houder.burgerservicenummer |
+
+    Scenario: persoonslijst heeft opschorting bijhouding reden "W" en zelfde reisdocumentnummer is gebruikt op andere persoonslijst
```

Comment:
ik neem aan dat dit alleen mogelijk is omdat één van de persoonslijsten is opgeschort met reden 'W'. Een reisdocument kan dus exact één houder hebben

Reply:
inderdaad